edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-cluster-maintenance.adoc

```diff
@@ -1,21 +0,0 @@
-:_mod-docs-content-type: ASSEMBLY
-[id="telco-troubleshooting-cluster-maintenance"]
-= Cluster maintenance
-include::_attributes/common-attributes.adoc[]
-:context: telco-troubleshooting-cluster-maintenance
-
-toc::[]
-
-In telco networks, you must pay more attention to certain configurations due the nature of bare-metal deployments.
-You can troubleshoot more effectively by completing these tasks:
-
-* Monitor for failed or failing hardware components
-* Periodically check the status of the cluster Operators
-
-[NOTE]
-====
-For hardware monitoring, contact your hardware vendor to find the appropriate logging tool for your specific hardware.
-====
-
-include::modules/telco-troubleshooting-clusters-check-cluster-operators.adoc[leveloffset=+1]
-include::modules/telco-troubleshooting-clusters-check-for-failed-pods.adoc[leveloffset=+1]
```

Comment:
Not deleted, just renamed: https://github.com/openshift/openshift-docs/pull/96100/files#diff-e0eb58922acf734883a6bb436240f5c31704ce2a02c0b1543652f8508e441cf8